base/deprecated.jl

```diff
@@ -443,3 +443,16 @@ export float32_isvalid, float64_isvalid
 @deprecate (&)(x::Char, y::Char)  Char(UInt32(x) & UInt32(y))
 @deprecate (|)(x::Char, y::Char)  Char(UInt32(x) | UInt32(y))
 @deprecate ($)(x::Char, y::Char)  Char(UInt32(x) $ UInt32(y))
+
+# 11241
+
+@deprecate is_valid_char(ch::Char) isvalid(ch)
+@deprecate is_valid_char(ch::Union(Unsigned, Integer)) isvalid(Char, ch)
```

Comment:
@ScottPJones, if you have
```
@deprecate is_valid_char(ch::Char) isvalid(ch)
@deprecate is_valid_char(ch) isvalid(Char, ch)
```
then it will use the single-argument form when `ch` is a `Char` and the two-argument form elsewhere.